src/util/color/predefinedcolorpalettes.cpp

```diff
@@ -390,6 +469,96 @@ const ColorPalette PredefinedColorPalettes::kMixxxKeyColorPalette =
                 });
 
+const ColorPalette PredefinedColorPalettes::kTraktorKeyColorPalette =
+        ColorPalette(
+                QStringLiteral("Traktor Key Colors"),
+                {
+                        kTraktorKeyColor1,
+                        kTraktorKeyColor2,
+                        kTraktorKeyColor3,
+                        kTraktorKeyColor4,
+                        kTraktorKeyColor5,
+                        kTraktorKeyColor6,
+                        kTraktorKeyColor7,
+                        kTraktorKeyColor8,
+                        kTraktorKeyColor9,
+                        kTraktorKeyColor10,
+                        kTraktorKeyColor11,
+                        kTraktorKeyColor12,
+                });
+
+const ColorPalette PredefinedColorPalettes::kMIKKeyColorPalette =
+        ColorPalette(
+                QStringLiteral("Mixed In Key - Key Colors"),
+                {
+                        kMIKKeyColor1,
+                        kMIKKeyColor2,
+                        kMIKKeyColor3,
+                        kMIKKeyColor4,
+                        kMIKKeyColor5,
+                        kMIKKeyColor6,
+                        kMIKKeyColor7,
+                        kMIKKeyColor8,
+                        kMIKKeyColor9,
+                        kMIKKeyColor10,
+                        kMIKKeyColor11,
+                        kMIKKeyColor12,
+                });
+
+const ColorPalette PredefinedColorPalettes::kProtKeyColorPalette =
+        ColorPalette(
+                QStringLiteral("Protanopia / Protanomaly Key Colors"),
+                {
+                        kProtKeyColor1,
+                        kProtKeyColor2,
+                        kProtKeyColor3,
+                        kProtKeyColor4,
+                        kProtKeyColor5,
+                        kProtKeyColor6,
+                        kProtKeyColor7,
+                        kProtKeyColor8,
+                        kProtKeyColor9,
+                        kProtKeyColor10,
+                        kProtKeyColor11,
+                        kProtKeyColor12,
+                });
+
+const ColorPalette PredefinedColorPalettes::kDeutKeyColorPalette =
+        ColorPalette(
+                QStringLiteral("Deuteranopia / Deuteranomaly Key Colors"),
+                {
+                        kDeutKeyColor1,
+                        kDeutKeyColor2,
+                        kDeutKeyColor3,
+                        kDeutKeyColor4,
+                        kDeutKeyColor5,
+                        kDeutKeyColor6,
+                        kDeutKeyColor7,
+                        kDeutKeyColor8,
+                        kDeutKeyColor9,
+                        kDeutKeyColor10,
+                        kDeutKeyColor11,
+                        kDeutKeyColor12,
+                });
+
+const ColorPalette PredefinedColorPalettes::kTritKeyColorPalette =
+        ColorPalette(
+                QStringLiteral("Tritanopia / Tritanomaly Key Colors"),
```

Comment:
It would be nice to have the user visible strings translatable. However make sure that the untranslated string is written to the preferences. 

You can use. QT_TR_NOOP() to use the same string translated and untranslated:  https://doc.qt.io/qt-5/i18n-source-translation.html#using-qt-tr-noop-and-qt-translate-noop-in-c

Reply:
The same applies to all pallets.